Dangerfile

```diff
@@ -13,16 +13,18 @@ avoid_exact_words = [
   { word: 'react native', reason: 'Please use React Native with capitals' }
 ]
 
-active_files = (git.modified_files + git.added_files)
+active_files = (git.modified_files + git.added_files).uniq
```

Comment:
This is surprising! Does the same file appear in both `git.modified_files` and `git.added_files`?

Reply:
Ha, it surprised us too! The PR included a commit that added the file, and subsequent commits that modified it. Without this line, we got duplicate failures.

Reply:
Ahh! Interesting! :100: